test/controllers/publishers_controller_test.rb

```diff
@@ -519,7 +515,7 @@ def request_login_email_uppercase_email(publisher:)
     assert_response 200
 
     publisher.reload
-    refute publisher.uphold_connection.uphold_verified?
+    refute publisher.uphold_connection
```

Comment:
This is the real test change. The rest was Rubocop formatting the file, which was apparently not in good shape.